pkg/node/node.go

```diff
@@ -479,9 +479,9 @@ func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, netwo
 			libp2pPrivateKey,
 			o,
 		)
-		logger.Infof("bootstrapper done, took %s", time.Since(start))
+		logger.Infof("bootstrapper took %s", time.Since(start))
```

Comment:
Should this be bootstrapper or BatchState initialization?

Reply:
I tend to agree that `bootstrapper` is hard to understand without context. My suggestion would be `postage snapshot syncing took %` because that would be consistent how the feature is called in the config (`use-postage-snapshot`).